app/services/facets/tagging_update_publisher.rb

```diff
@@ -80,6 +81,12 @@ def generate_links_payload
       }
     end
 
+    # FIXME: This is a temporary tag set which can be removed once
+    # we've updated finder email signup to handle links based signup config.
+    def email_alert_links_payload
```

Comment:
Nitpick: This should probably be called `email_alert_tags_payload`

Reply:
Good point, I've amended.